pdu/ShortMessage.go

```diff
@@ -84,6 +111,25 @@ func (c *ShortMessage) SetMessageData(data []byte) {
 	c.messageData = data
 }
 
+// GetMessageData returns underlying binary message.
+func (c *ShortMessage) GetMessageData() (data []byte, err error) {
```

Comment:
I think this is good for some ad-hoc use case, should be added to the codebase..